File: src/main/java/com/googongill/aditory/controller/dto/link/DeleteLinkResponse.java
package com.googongill.aditory.controller.dto.link;

import com.fasterxml.jackson.annotation.JsonAutoDetect;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import lombok.Builder;

@Builder
@JsonSerialize
@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
public class DeleteLinkResponse {
    private Long linkId;

    public static DeleteLinkResponse of(Long linkId) {
        return DeleteLinkResponse.builder()
                .linkId(linkId)
                .build();
    }
}

File: src/main/java/com/googongill/aditory/controller/dto/link/response/DeleteLinkResponse.kt
package com.googongill.aditory.controller.dto.link.response

import com.fasterxml.jackson.annotation.JsonAutoDetect
import com.fasterxml.jackson.databind.annotation.JsonSerialize

@JsonSerialize
@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
data class DeleteLinkResponse(
    val linkId: Long?
) {

    companion object {
        fun of(linkId: Long?): DeleteLinkResponse {
            return DeleteLinkResponse(
                linkId = linkId
            )
        }
    }
}
